definitions/npm/react-native-audio-toolkit_v1.x.x/flow_v0.32.x-/react-native-audio-toolkit_v1.x.x.js

```diff
@@ -0,0 +1,58 @@
+type $npm$react_native_audio_toolkit$PlaybackOptions = {
```

Comment:
Why are these types declared in the global and not in the `declare module` scope?